pkg/apiserver/rest/apis/v1/types.go

```diff
@@ -464,6 +464,24 @@ type DockerHubRepository struct {
 	Status          string `json:"status"`
 }
 
+// HandleApplicationTriggerJFrogRequest application trigger JFrog webhook request
+type HandleApplicationTriggerJFrogRequest struct {
+	Domain    string           `json:"domain"`
+	EventType string           `json:"event_type"`
+	Data      JFrogWebhookData `json:"data"`
+}
+
+// JFrogWebhookData is the data of JFrog webhook request
+type JFrogWebhookData struct {
+	URL       string
```

Comment:
JSON tag?

Reply:
it's not parse from request body, i am not sure need json tag.